src/statsig/constants.ts

```diff
@@ -53,6 +53,7 @@ export const ExperimentConfigs = {
     defaultValues: {
       onboardingNameScreenEnabled: true,
       chooseAdventureEnabled: false,
+      cashInBottomSheetEnabled: true,
```

Comment:
this should not have been removed to begin with

Reply:
even if it wasn't used in wallet code anymore, we need to give defaults for all experiment params when we query for them from statsig

Reply:
i thought the cash in bottom sheet is a remote config and had nothing to do with statsig....why do we need this here? i couldn't find anything related to cash in bottom sheet in statsig